src/main/kotlin/me/owdding/customscoreboard/feature/customscoreboard/CustomScoreboardRenderer.kt

```diff
@@ -46,6 +50,18 @@ object CustomScoreboardRenderer {
         updateDisplay()
     }
 
+    @Subscription
+    fun onScreenOpen(event: ScreenMouseClickEvent.Pre) {
```

Comment:
pre is ran before containers click and since this will be rendered under everything containers clicks should go first so post I think would be better, does require PR to api to add result to post tho